server/sql/product_answer.sql

```diff
@@ -0,0 +1,19 @@
+--Create product_answer table
+DROP TABLE IF EXISTS product_answer;
+
+CREATE TABLE product_answer (
+  id SERIAL PRIMARY KEY,
+  question_id INT,
+  body VARCHAR(255) NOT NULL,
+  date_written BIGINT NOT NULL,
+  answerer_name VARCHAR(50) NOT NULL,
+  answerer_email VARCHAR(255) NOT NULL,
+  reported BOOLEAN NOT NULL,
+  helpful INTEGER NOT NULL,
+  CONSTRAINT fk_questionid
+    FOREIGN KEY(question_id)
+      REFERENCES product_question(id)
+        ON DELETE CASCADE
```

Comment:
This is a really clever way to handle deletions. I was writing `CASCADE` on every delete command instead of including it in the actual table 🤣